addons/csi-azuredisk/csi-azuredisk-node.yaml

```diff
@@ -90,6 +90,8 @@ spec:
             - "--nodeid=$(KUBE_NODE_NAME)"
             - "--metrics-address=0.0.0.0:29605"
             - "--enable-perf-optimization=true"
+            - "--allow-empty-cloud-config=true"
```

Comment:
in azure-disk-controller this flag is false, is this correct here (or there)?

Reply:
This is coming from the upstream manifest. I believe it's intentional. It might be very valid that you don't need cloud-config for the node-controller, but that you need it for the disk-controller.